docs/clients/grpc/projections/README.md

```diff
@@ -0,0 +1,665 @@
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+updateProjection(projectionName: string,query: string,options?: UpdateProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+Updating a projection that does not exists results in an error:
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#Update_NotFound
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+updateProjection(projectionName: string,query: string,options?: UpdateProjectionOptions): Promise<void>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## List all projections
+
+Returns a list of all projections.
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#ListAll
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```
+// This is currently not available in the NodeJS client
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## List continuous projections
+
+Returns a list of all continuous projections
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#ListContinuous
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+listContinuousProjections(options?: ListProjectionsOptions): Promise<ProjectionDetails[]>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Get Status 
+
+Gets the status of a named projection
+
+<xode-group>
+<xode-block title="C#">
+
+<<< @/docs/clients/dotnet/generated/main/samples/projection-management/Program.cs#GetStatus
+
+</xode-block>
+<xode-block title="Java">
+
+```java
+// This is currently not available in the java client
+```
+</xode-block>
+<xode-block title="NodeJS">
+
+```ts
+ getProjectionStatistics(projectionName: string,options?: GetProjectionStatisticsOptions): Promise<ProjectionDetails>;
+```
+</xode-block>
+<xode-block title="Rust">
+
+```Rust
+// This is currently not available in the Rust client
+```
+</xode-block>
+</xode-group>
+
+## Get state
+
+Retrieves the state of a projection.
```

Comment:
**SUGGESTION:** Would be good to add some context to this operation (the same apply to the methods below). Especially what's the state in fact.

Reply:
That is explained in the server doc linked at the very top of the doc.

Reply:
Yup, but at least a sentence what's the state, in fact, would be good.